README.md

```diff
@@ -31,7 +31,12 @@ The docker image is available
 > * CPU: >= 6 cores
 > * Memory: 25~30G
 > 
-> **2x2 tiles run time**: ~ 40 hours (clk_period=1000)
+> **2x2 tiles run time**:  
+> | clk_period (ps) | frequency (Hz) | time (hour) |  
+> |-------|-------|-------|  
+> | 1000            | 1G             | ~40         |
+> | 10_000          | 100M           | ~7          |  
```

Comment:
Can we replace `10_000` with `10, 000`?

Reply:
Move to https://github.com/coredac/CGRA-Flow/pull/1